Scripts/fastlane/Deliverfile

```diff
@@ -11,7 +11,7 @@ privacy_url({
   'default' => 'https://automattic.com/privacy/',
 })
 
-copyright('2020 Automattic Inc.')
+copyright('2021 Automattic Inc.')
```

Comment:
👍 
Made me realize that for Woo I updated the value in ADC but didn't think about updating it in Deliverfile

Reply:
Oh weird – `fastlane deliver` yelled at me in the `precheck` stage about this, which was how I figured out that it was in the `deliverfile` at all 😄 

Personally I'd like to get rid of the deliver file and just put these in .txt files instead but 🤷‍♂️

Reply:
Yeah `fastlane deliver` also yelled at me for it which is what prompted me asking about it to Legal; but at that point I thought it was pulling the data from ADC and putting it into the `copyright.txt` file and that's where the warning was coming from, which is why I updated the info in ADC directly, thinking it was pulled from the `.txt`.

I didn't know we also mentioned it in the `Deliverfile`, and I wonder if it's still useful to keep it given that we also have it in the `copyright.txt` file anyway? 🤔

Reply:
Yeah for WPiOS it's specified in the `deliverfile` which is fine, but given that all our other metadata is stored in the fastlane-specified `.txt` files IMHO it'd be nice to have that be consistent.

Might open a PR to that effect later today